packages/config/config/devices/0x0129/cp2-5-3_1.78.json

```diff
@@ -85,97 +84,101 @@
 			"allowManualEntry": true
 		},
 		"7": {
-			"label": "Shut Down Time (after wrong code entries)",
-			"unit": "Seconds",
+			"label": "Shutdown Time",
+			"description": "Number of times the device is locked out after the invalid retries are exceeded",
 			"valueSize": 1,
-			"minValue": 10,
-			"maxValue": 127,
-			"defaultValue": 10,
+			"minValue": 1,
+			"maxValue": 255,
+			"defaultValue": 30,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"8": {
 			"label": "Operating Mode",
+			"description": "Sets if the unit operates normal, or disables keypad and Z-Wave lock/unlock",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 2,
+			"maxValue": 3,
 			"defaultValue": 0,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Normal Mode",
+					"label": "Normal",
 					"value": 0
 				},
 				{
-					"label": "Vacation Mode (Keypad Locked Out)",
+					"label": "Vacation Mode",
 					"value": 1
 				},
 				{
-					"label": "Privacy Mode (Keypad Locked, RF Unlock Functional)",
-					"value": 2
+					"label": "Privacy Mode",
+					"value": 3
 				}
 			]
 		},
 		"11": {
 			"label": "One Touch Locking",
 			"valueSize": 1,
-			"minValue": -128,
-			"maxValue": 0,
-			"defaultValue": 0,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "One Touch Locking On",
-					"value": -128
+					"label": "Disable",
+					"value": 0
 				},
 				{
-					"label": "One Touch Locking Off",
-					"value": 0
+					"label": "Enable",
+					"value": 255
 				}
 			]
 		},
 		"13": {
 			"label": "Lock Status LED",
 			"valueSize": 1,
-			"minValue": -128,
-			"maxValue": 0,
+			"minValue": 0,
+			"maxValue": 255,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED On",
-					"value": -128
+					"label": "Disable",
+					"value": 0
 				},
 				{
-					"label": "LED Off",
-					"value": 0
+					"label": "Enable",
+					"value": 255
 				}
 			]
 		},
 		"15": {
 			"label": "Reset to Factory Defaults",
 			"valueSize": 1,
-			"minValue": -128,
-			"maxValue": 0,
+			"minValue": 0,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Reset to Factory On",
-					"value": -128
+					"label": "Normal Operation",
+					"value": 0
 				},
 				{
-					"label": "Reset to Factory Off",
-					"value": 0
+					"label": "Reset Device",
+					"value": 1
 				}
```

Comment:
dito

Reply:
This also appears to have been wrong in the files:

![Screen Shot 2021-02-08 at 9 59 47 PM](https://user-images.githubusercontent.com/33612110/107309719-f67d3880-6a58-11eb-8f4a-8e224e37a963.png)